src/main/java/frc/robot/subsystems/SwerveDriveTrainSubsystem.java

```diff
@@ -94,7 +96,7 @@ public SwerveDriveTrainSubsystem() {
   @Override
   public void periodic() {
     // This method will be called once per scheduler run
-    var gyroAngle = mGyro.getRotation2d();
+    gyroAngle = mGyro.getRotation2d();
```

Comment:
Please rename `gyroAngle` to `mGyroAngle` to indicate that it is a class member.